code/modules/clothing/suits/toggles.dm

```diff
@@ -59,7 +58,6 @@
 			else
 				H.equip_to_slot_if_possible(hood,slot_head,0,0,1)
 				suittoggled = 1
-				src.icon_state = "[initial(icon_state)]_t"
```

Comment:
The icons are used, wintercoats it to close the 1-pixel gap when the suit is toggled to be closed.
See also the ian and carp suits, chaplain hoodie, and all the other icons that end in "_t" in mob/suit.dmi)